modules/icinga/manifests/check_feature.pp

```diff
@@ -0,0 +1,14 @@
+# FIXME: This class needs better documentation as per https://docs.puppetlabs.com/guides/style_guide.html#puppet-doc
+define icinga::check_feature(
+  $feature,
+  $ensure = present,
+  $notes_url = undef
+) {
+  icinga::check { "smokey_loop_for_${feature}":
```

Comment:
I was following along until this last commit, where the commit message doesn't really describe what the commit is doing.

I get that `modules/icinga/manifests/check_feature.pp` was once doing one thing, then was renamed and edited to do something else, and now we've re-created `modules/icinga/manifests/check_feature.pp` to do something again. It would be good to have a paragraph in this last commit to explain how/why we've come full circle.

Thanks!

Reply:
Oh dear, you're right to be confused - I got in a tangle with the renaming. The new commit is a one-liner; could you take another look?